src/app/components/ATIAnalytics/atiUrl/index.ts

```diff
@@ -499,14 +499,19 @@ export const buildReverbPageSectionEventModel = ({
   advertiserID,
   url,
 }: ATIEventTrackingProps) => {
-  const eventPublisher = type === 'view' ? 'ati' : 'atc';
-
   const eventDetails = {
     eventName: type === 'view' ? 'sectionView' : 'sectionClick',
-    ...(type === 'click' && {
+    ...{
+      ...{ eventPublisher: type === 'click' ? 'click' : 'impression' },
```

Comment:
The double spreading here looks a little strange, is that intentional?

Reply:
Good spot. This is an oversight from moving code from another place.